runtime/src/anchor/mod.rs

```diff
@@ -204,7 +204,7 @@ decl_module! {
             };
 
             let anchor_data_encoded = anchor_data.encode();
-            runtime_io::set_child_storage(&child_storage_key, anchor_id.as_ref(), &anchor_data_encoded);
+            runtime_io::storage::child_set(&child_storage_key, anchor_id.as_ref(), &anchor_data_encoded);
```

Comment:
does this mean that the newer version has changed the file/package structure?

Reply:
Exactly 👍